frontend/scripts/setup/ubuntu_setup.sh

```
@@ -0,0 +1,121 @@
+  echo_info "Uninstalling Android SDK..."
+  if ! sudo apt-get remove --purge -y android-sdk; then
+    echo_error "Failed to uninstall Android SDK."
+    exit 1
+  fi
+
+  echo_info "Cleaning up configuration files..."
+  if ! sed -i '/flutter\/bin/d' ~/.bashrc || ! sed -i '/Android\/Sdk/d' ~/.bashrc; then
+    echo_error "Failed to clean up configuration files."
+    exit 1
+  fi
+  source ~/.bashrc
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)